include/openenclave/bits/result.h

```diff
@@ -300,24 +300,19 @@ typedef enum _oe_result
     OE_QUOTE_ENCLAVE_IDENTITY_VERIFICATION_FAILED,
 
     /**
-     * QE identify mrsigner do not match.
+     * QE identify unique Id do not match.
```

Comment:
Copyedit: Clean up the grammar and spelling for the comments e.g.  "identify" should be "identity". I would suggest:

- Failed to verify the identity of the quoting enclave.
- Unique ID of the quoting enclave does not match the expected value.
- Product ID of the quoting enclave does not match the expected value.
- AES CMAC of the report does not match the expected value.

Reply:
Done.